arch/arm/mach-imx/Kconfig

```diff
@@ -87,6 +87,14 @@ config SPL_IMX_HAB
 	  This option enables the support for secure boot (HAB) in SPL.
 	  See doc/README.mxc_hab for more details.
 
+config IMX_HAB_IMAGE_AUTH
```

Comment:
I would rather see this commit be submitted upstream.  In general, this is not our use-case as we avoid the entire code path by not enabling `IMX_OPTEE`.

Reply:
And we don't use `booti` / `bootz` code paths.